src/qtt/instrument_drivers/virtualAwg/virtual_awg.py

```diff
@@ -182,15 +182,19 @@ def disable_outputs(self, gate_names):
             A start command is required to enable the outputs.
 
         Arguments:
-            gate_names (list) The names of the gates which needs to be disabled.
+            gate_names (list or None) The names of the gates which needs to be disabled. If None, then disable all gates
```

Comment:
Make argument gate_names optional and defaults to gate_names=None, then we don't have to call with parameter None.
Then it is also consistent with the disable_output implementations of the instruments itself